core/domain/feedback_jobs_continuous.py

```diff
@@ -135,26 +149,25 @@ def get_thread_analytics_multi(cls, exploration_ids):
                 exploration_ids))
 
         return [feedback_domain.FeedbackAnalytics(
-            exploration_ids[i],
-            (realtime_models[i].num_open_threads
-             if realtime_models[i] is not None else 0) +
+            exploration_ids[i], (realtime_models[i].num_open_threads
```

Comment:
C:153, 0: Wrong continued indentation (add 20 spaces).
             if realtime_models[i] is not None else 0) +
             ^                   | (bad-continuation)

It's not coming up currently while I have the old styling however, not sure why it was before.